app/setup.py

```diff
@@ -341,6 +349,21 @@ def setup_publisher(application):
         raise Exception("Unknown EQ_PUBLISHER_BACKEND")
 
 
+def setup_feedback(application):
+    if application.config["EQ_FEEDBACK_BACKEND"] == "gcs":
+        bucket_id = application.config.get("EQ_GCS_FEEDBACK_BUCKET_ID")
+
+        if not bucket_id:
+            raise Exception("Setting EQ_GCS_FEEDBACK_BUCKET_ID Missing")
+
+        application.eq["feedback"] = GCSFeedback(bucket_name=bucket_id)
```

Comment:
I think it would be clearer if this was:
```suggestion
        application.eq["feedback_submitter"] = GCSFeedback(bucket_name=bucket_id)
```